packages/grid/src/grid.ts

```diff
@@ -0,0 +1,50 @@
+import type { CSSResultGroup, TemplateResult } from 'lit';
+import { virtualize } from '@lit-labs/virtualizer/virtualize.js';
+import { LitElement, html } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import styles from './grid.scss.js';
+import { GridColumn } from './column.js';
+
+export class Grid<T extends { [x: string]: unknown } = Record<string, unknown>> extends LitElement {
+  /** @private */
+  static override styles: CSSResultGroup = styles;
+
+  @state() columns: Array<GridColumn<T>> = [];
+
+  @property() items: T[] = [];
+
+  @property({ type: Boolean, reflect: true, attribute: 'no-border' }) noBorder?: boolean;
+
+  @property({ type: Boolean, reflect: true, attribute: 'no-row-border' }) noRowBorder?: boolean;
+
+  render(): TemplateResult {
+    return html`
+      <slot @slotchange=${this.#onSlotchange} style="display:none"></slot>
+      <table>
+        <thead>
+          <tr>
+            ${this.columns.map(col => col.renderHeaderCell())}
+          </tr>
+        </thead>
+        <tbody>
+          ${virtualize({ items: this.items, renderItem: item => this.renderItem(item) })}
+        </tbody>
+        <tfoot></tfoot>
```

Comment:
Is `tfoot` necessary here since we don't add anything inside?

Reply:
It's a placeholder for now.